users/services/identity.py

```diff
@@ -58,7 +58,10 @@ def admin_identities(cls) -> models.QuerySet[Identity]:
 
     def following(self) -> models.QuerySet[Identity]:
         return (
-            Identity.objects.filter(inbound_follows__source=self.identity)
+            Identity.objects.filter(
+                inbound_follows__source=self.identity,
+                inbound_follows__state=FollowStates.accepted,
```

Comment:
Could you change this to be `__in=FollowStates.group_active()`? I think including pending follows makes sense.

Reply:
Sure, just updated it :slightly_smiling_face: